core/src/components/range/range.tsx

```diff
@@ -43,6 +44,7 @@ import type { PinFormatter } from './range-interface';
 @Component({
   tag: 'ion-range',
   styleUrls: {
+    base: 'range.scss',
```

Comment:
Same situation as checkbox.

Reply:
Unclear on what to do with this one. We could "fix" it by adding to the base styles:

```css
  --bar-height: 2px;
  --bar-background: #ddd;
  --height: 10px;
  --knob-handle-size: 10px;
  --knob-size: 10px;
  --knob-background: #eee;
```

But developers then need to override each of those variables. Similar to the checkbox/radio we don't extend the default `<input type="range" />` element, we do something custom. Thoughts?

Reply:
I'm okay with letting developer feedback roll in first and leaving it as-is for now.